kornia/augmentation/container/augment.py

```diff
@@ -276,18 +277,23 @@ def inverse(  # type: ignore[override]
             else:
                 outputs[idx] = input
 
-        _outputs = [i for i in outputs if isinstance(i, Tensor)]
+        _outputs = [i for i in outputs if isinstance(i, Tensor) or KORNIA_CHECK_IS_LIST_OF_TENSOR(i)]
```

Comment:
I am thinking of more. So that if some users input a `Box` object directly. We may output the modified `Box` object. I think we shall remove this and do some consistency check between the inputs and outputs instead. Potentially, we may support TV's input objects as well.

Reply:
totally agree here, but we can ship this pr to fix the bug, and after work on this approach? -- probably this will be easy to be done after we have the new BBox API

Reply:
Sure thing.